src/ReactiveUI.Validation.AndroidX/ReactiveUI.Validation.AndroidX.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>MonoAndroid90</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <Nullable>enable</Nullable>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.*" />
```

Comment:
Where's this material coming from btw? We have ReactiveUI.AndroidX

Reply:
That `Xamarin.Google.Android.Material` is required for the `TextInputLayout` control used in [`BindValidation`](https://github.com/reactiveui/ReactiveUI.Validation/blob/androidx/src/ReactiveUI.Validation.AndroidSupport/Extensions/ViewForExtensions.cs#L47). That package doesn't seem to be included in `ReactiveUI.AndroidSupport`. According to https://developer.android.com/jetpack/androidx/migrate/class-mappings the `TextInputLayout` control from `Xamarin.Android.Support.Design` now lives in `Xamarin.Google.Android.Material` that is an AndroidX equivalent for the `Xamarin.Android.Support.Design` package.

Reply:
Probably we should reference `ReactiveUI.AndroidX` and `ReactiveUI.AndroidSupport` here and there anyway?